flink-ai-flow/examples/wide_and_deep/dependencies/python/code/project.yaml

```diff
@@ -1,6 +1,6 @@
 project_name: wide_and_deep
 server_ip: localhost
 server_port: 50051
-notification_uri: localhost:50052
+notification_uri: localhost:50051
```

Comment:
Will the wdl with jar also want users to use example server? Or using the `start-aiflow.sh` script but using embedded notification service?

Reply:
`start-aiflow.sh` has been renamed to `start-all-aiflow-services.sh`, which will use the embedded notification service/